packages/video_player/video_player/lib/video_player.dart

```diff
@@ -726,20 +729,46 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   ///
   /// If no [closedCaptionFile] was specified, this will always return an empty
   /// [Caption].
+
   Caption _getCaptionAt(Duration position) {
-    if (_closedCaptionFile == null) {
+    if (_closedCaptionFile == null || _sortedCaptions == null) {
```

Comment:
Please add a `final List<Captions>? sortedCaptions = _sortedCaptions;` above this, and then use `sortedCaptions` everywhere, so that repeated force-unwraps aren't needed below.